proto/buf/connect/demo/eliza/v1/eliza.proto

```diff
@@ -50,3 +57,23 @@ message ConverseRequest {
 message ConverseResponse {
   string sentence = 1;
 }
+
+// IntroduceRequest describes the sentence said to the ELIZA program.
+message IntroduceRequest {
+  string sentence = 1;
+}
+
+// IntroduceResponse describes the sentence responded by the ELIZA program.
+message IntroduceResponse {
+  string sentence = 1;
+}
+
+// AskRequest describes the sentence said to the ELIZA program.
+message AskRequest {
+  string sentence = 1;
```

Comment:
Should this be repeated so we can send a bunch of sentences for Eliza to slowly respond to one at a time?

Reply:
Yeah maybe.  I think it's probably OK for now just to have a single sentence.  We can modify it if just the singular field doesn't work well.